project-p1-first-webpage-master/code/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
```

Comment:
The folders structure is a bit confusing, but I understood that the relevant and more recent code what the one in the "project-p1-first-webpage-master" folder, so I reviewed only starting with this file.